.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,18 @@
+name: Tutor Integration Tests
+on: [pull_request]
+
+jobs:
+  integration-test:
```

Comment:
Can we name the job so `integration-test` is not used as default? We can use the same `Tutor Integration Test` so it looks better here: https://github.com/eduNEXT/eox-hooks/actions/runs/9392586481/job/25867080058?pr=52. It'd look duplicated in the CI checks, but I don't think that would be a problem. See examples here: 
![image](https://github.com/eduNEXT/eox-hooks/assets/64440265/b1d132ac-b154-4b90-9602-2117cc8908a3)

Reply:
Done, thanks